justspaces/local_settings.example.py

```diff
@@ -13,7 +13,7 @@
     'default': {
         'ENGINE': 'django.contrib.gis.db.backends.postgis',
         'NAME': 'just-spaces',
-        'USER': '',
+        'USER': 'postgres',
```

Comment:
This might change depending on my comment above, i.e., you  might be able to leave it as a blank string.